geany.nsi.in

```diff
@@ -124,13 +125,17 @@ Section "!Program Files" SEC01
 
 	SetOutPath "$INSTDIR\bin"
 	File "${RESOURCEDIR}\bin\Geany.exe"
-	File "${RESOURCEDIR}\bin\Geany*.dll"
+	File "${RESOURCEDIR}\bin\*Geany*.dll"
```

Comment:
Yes. waf geany-0.dll and autotools libgeany-0.dll. The linker accepts both for -lgeany.